dts/msm8974/msm8974-lge-hammerhead.dts

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <0x7e 0x96 0x20002 0x0b>;
+
+	hammerhead {
+		model = "LG Google Nexus 5";
+		compatible = "lge,hammerhead", "qcom,msm8974", "lk2nd,device";
+		lk2nd,match-bootloader = "HHZ20h";
+
+		lk2nd,keys =
+			<KEY_VOLUMEUP   PM_GPIO(2) PM_GPIO_PULL_UP_1_5>,
+			<KEY_VOLUMEDOWN PM_GPIO(3) PM_GPIO_PULL_UP_1_5>;
```

Comment:
do you really need these keys definition? Defaults don't work?..

Reply:
Yes this is needed. Without these lines volume buttons don't work.

Reply:
ok, at least gpio2&3 and pull-up matches upstream dts for hammerhead, so I guess it's fine..